data/device-tests/jabra-evolve2-75.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Jabra Evolve2 75",
+  "interactive": false,
+  "steps": [
+    {
+      "url": "https://fwupd.org/downloads/335f10696cd57ae74b2115a9e44a39a2c5208e7e090d1f4e72bb91bc65a7480d-jabra-evolve2-75-1.8.13.cab",
+      "components": [
+        {
+          "version": "1.8.13",
+          "protocol": "com.jabra.gnp",
+          "guids": [
+            "354b39e6-e12e-557e-8beb-fa4128b0a98c",
+            "dc636868-13f1-5c3b-bac5-c4432d447707",
+            "b0870be4-7c69-50af-9b7d-fdfa8d849607"
```

Comment:
just to check -- three guids?

Reply:
Yes. The device has 3 different PIDs, which translates to 3 GUIDs being supported by the firmware. 
Unless I'm misunderstanding something

Reply:
Right, all good. I was just checking you weren't using the GUID for `USB\VID_1234` for instance.